packages/iov-multichain/src/signingservercore.ts

```diff
@@ -57,17 +57,20 @@ export class SigningServerCore {
   private readonly authorizeGetIdentities: GetIdentitiesAuthorization;
   private readonly authorizeSignAndPost: SignAndPostAuthorization;
   private readonly signedAndPostedProducer = new DefaultValueProducer<readonly SignedAndPosted[]>([]);
+  private readonly logError: (error: any) => void;
```

Comment:
Couldn't we make the `error` an `Error`?

Reply:
Good idea, but …